ac-58 Create fake data for traders, farmers, and product transactions.

ExtraGravity opened the pull request:
Created 3 traders with usernames and passwords matching their names.
Created 8 farmers.
Created 20 fully randomized product transactions.
The product transactions created dates are guaranteed to be before the last modified date.

ExtraGravity committed:
ac-58 Create fake data for traders, farmers, and product transactions.

Created 3 traders with usernames and passwords matching their names.
Created 8 farmers.
Created 20 fully randomized product transactions.
The product transactions created dates are guaranteed to be before the last modified date.

beaesguerra reviewed (approved):
looks good, merge when the mock-data script has run and worked properly :)

ExtraGravity committed:
ac-58 add fix quotations on brian

ExtraGravity committed:
Merge branch 'master' into enoch/ac-58-mock-product-transactions-data

ExtraGravity committed:
ac-55 fix mock-data, integrate to dockerize script, update db schema

Some commas were were on the last line of queries in the mock-data.
Quotes were not escaped properly.
dockerize scripts runs init-db.sql then mock-data.sql.
Units for milk density field are acutally unknown, so changing to milkQuality.
